src/macros-by-example.md

```diff
@@ -76,6 +76,8 @@ delimiters for the matcher will match any pair of delimiters. Thus, for
 instance, the matcher `(())` will match `{()}` but not `{{}}`. The character
 `$` cannot be matched or transcribed literally.
 
+#### Forwarding a matched fragment
```

Comment:
I think it's fine to use a 3rd level header. We generally try to keep the headers nested in a natural order where h4 is nested in an h3 which is nested in an h2 which is nested in an h1. Jumping the order can break that hierarchy.

```suggestion
### Forwarding a matched fragment
```